data/weapons/weapon_data.csv

```diff
@@ -1,59 +1,59 @@
+Opfer GV Suppression Artillery,diableavionics_opfergv,3,0.5,8000,900,,10,,20,12,20,320,25,160,KINETIC,9,,0.5,0.75,9999,0.075,2,10,0.05,4,,800,,,25,,,,"kinetic18, diable_weapon_bp, rare_bp",,Diable Avionics,,Shield Suppression,,,,,,,,,,
 ,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
 ,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
 ,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
 Winee EMPR,diableavionics_winee,2,0.5,600,600,,25,400,25,50,7,,,,ENERGY,50,,0.5,7.2,5,0.075,,,,,,1000,,,25,,,FIRE_WHEN_INEFFICIENT,"energy5, diable_weapon_bp, rare_bp",,Diable Avionics,,Support,,,,,"%s: Hits on shields while at high hard flux or on hull have a %s chance to arc to weapons and engines, dealing extra EMP damage. ",EMP Arcs|25%,,,,
 ,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
-Burchel Beam Repeater,diableavionics_burchel,1,0.5,600,600,250,,,,75,8,6,0.5,3,ENERGY,,200,,,0.48,0.02,,,,,1000000,,,,,,,"PD,ANTI_FTR","pd9,beam8, diable_weapon_bp, rare_bp",,Diable Avionics,,Point Defense,,,,,,,,,,
-Glaux Lance Repeater,diableavionics_glaux,2,0.5,2250,700,687,,,,16,11,6,0.605,2,ENERGY,,756,,,0.48,0.02,,,,,1000000,,,,,,,,"energy14, beam10, diable_weapon_bp, rare_bp",,Diable Avionics,,Strike,,,,,,,,,,
+Burchel Beam Repeater,diableavionics_burchel,1,0.5,600,600,300,,,,75,8,12,1,3,ENERGY,,240,,,0.48,0.02,,,,,1000000,,,,,,,"PD,ANTI_FTR,PD_ALSO","pd9,beam8, diable_weapon_bp, rare_bp",,Diable Avionics,,Point Defense,,,,,,,,,,
+Glaux Lance Repeater,diableavionics_glaux,2,0.5,2250,700,687,,,,16,11,8,0.727,2,ENERGY,,618,,,0.48,0.02,,,,,1000000,,,,,,,,"energy14, beam10, diable_weapon_bp, rare_bp",,Diable Avionics,,Strike,,,,,,,,,,
 ,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
-Hexafire Scatter-Cannon,diableavionics_hexafire,1,,1750,700,,100,30,10,15,11,,,,ENERGY,110,,0,2.5,,,2,16,0.6,6,,600,,,100,,,,"energy13, diable_weapon_bp, rare_bp",,Diable Avionics,,General,,,,,%s: projectiles deal up to %s extra damage the higher the firing ship's %s is.,Flux charged ordinance|50%| flux level,,,,
-Twin-Hexafire Scatter-Cannon,diableavionics_hexafire2,2,,5000,700,,100,30,10,8,25,,,,ENERGY,110,,0,2.5,,,2,16,0.3,6,,600,,,120,,,,"energy17, diable_weapon_bp, rare_bp",,Diable Avionics,,General,,,,,%s: projectiles deal up to %s extra damage the higher the firing ship's %s is.,Flux charged ordinance|50%|flux level,,,,
+Hexafire Scatter-Cannon,diableavionics_hexafire,1,,1750,700,,120,30,10,15,11,,,,ENERGY,120,,0,2.5,,,2,16,0.6,6,,1200,,,100,,,,"energy13, diable_weapon_bp, rare_bp",,Diable Avionics,,General,,,,,%s: projectiles deal up to %s extra damage the higher the firing ship's %s is.,Flux charged ordinance|100%| flux level,,,,
+Twin-Hexafire Scatter-Cannon,diableavionics_hexafire2,2,,5000,700,,120,30,10,8,25,,,,ENERGY,120,,0,2.5,,,2,16,0.3,6,,1200,,,120,,,,"energy17, diable_weapon_bp, rare_bp",,Diable Avionics,,General,,,,,%s: projectiles deal up to %s extra damage the higher the firing ship's %s is.,Flux charged ordinance|100%|flux level,,,,
```

Comment:
Hexafire cannon base damage increased to 120 (from 100), flux cost increased to 120 (from 110),  projectile speed doubled. The flux level damage buff cap is increased to 100% at max flux (from 50% at max flux).
Twin-Hexafire cannon received the same buffs, except that it still fires twice as much bullets.